src/content_script.js

```diff
@@ -74,6 +74,11 @@ const PASSIVE_SHARE_PATTERN_DETECTION_SELECTORS = [
   "[href*='facebook.com/sharer']", // Legacy Share dialog
 ];
 
+window.addEventListener("message", (e) => {
+  if (e.data === "closeTheInjectedIframe") {
+    closeIframe();
```

Comment:
One issue I had was figuring out how to enable cross-origin communication between the `iframe` embedded site and the parent page. I thought this was impossible according to my research until I found the [Window.postMessage() ](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) method! Super neat and useful to know for add on development.